### DISCUSSION TITLE
IntelliSense Suddenly Stopped Working With V4

### DISCUSSION THREAD

**Discussion by martin-ukhanov** [tailwindlabs/tailwindcss#15424] [Help]
Hello. The other day my IntelliSense stopped working out of the blue after using V4 for the past couple weeks with no issues.

If I create a fresh project that uses V3 the IntelliSense works, however after migrating to V4 it stops working again.

Has anyone else run into this or knows how to fix it? Thanks.

**Comment by devlzcode** [comment]
Try switching to the pre-release version if you're using VS Code.

**Reply by martin-ukhanov** [reply]
I am using the pre-release version.

**Comment by pvds** [comment]
@martin-ukhanov be aware these is an open bug: [Custom @import breaks Intellisense on v4 beta](https://github.com/tailwindlabs/tailwindcss-intellisense/issues/1103#top)

[This PR](https://github.com/tailwindlabs/tailwindcss-intellisense/pull/1106) has been opened but @thecrypticace mentioned it's not an easy fix.

**Comment by thecrypticace** [comment]
Hi @martin-ukhanov!

Can you open an issue with a reproduction [in the intellisense repo](https://github.com/tailwindlabs/tailwindcss-intellisense/issues) please?

**Reply by martin-ukhanov** [reply]
Yes, I'll be able to do that tomorrow.

**Reply by martin-ukhanov** [reply]
[https://github.com/tailwindlabs/tailwindcss-intellisense/issues/1115](https://github.com/tailwindlabs/tailwindcss-intellisense/issues/1115)
